hits_ingest.py

```diff
@@ -5,88 +5,102 @@
-1. repo, where the ingested images and registry will go
-2. calibrepo, where the calibration product registry will go
+This script processes raw decam images from ingestion --> difference imaging!
 
 USAGE
-$ python hits_ingest.py 'path/to/datadir/'
+$ python hits_ingest.py ingest -f path/to/rawimages/
+(will use all files in the directory named *.fz if no files are specified)
+or
+$ python hits_ingest.py ingestCalibs -f path/to/calibrations/somefiles*.fits.fz
 or
-$ python hits_ingest.py 'path/to/datadir/justsomefiles*.fits.fz'
-(the quotes around the datadir string are important!)
+$ python hits_ingest.py processCcd
 or
-$ python hits_ingest.py
-(if you don't want to give it an argument because you're running processCcd)
+$ python hits_ingest.py diffIm
 
 OUTPUT
-if doIngest: repo populated with *links* to files in datadir, organized by date
-             sqlite3 database registry of ingested images also created in repo
-if doIngestCalibs: sqlite3 database registry of ingested calibration products
-                   created in calibrepo
-if doProcessCcd: outputrepo/visit populated with subdirectories containing the
-                 usual post-ISR data (bkgd, calexp, icExp, icSrc, postISR)
-
-BASH EQUIVALENT
-if doIngest:
+ingest: repo populated with *links* to files in datadir, organized by date
+        sqlite3 database registry of ingested images also created in repo
+ingestCalibs: calibrepo populated with *links* to files in datadir,
+              organized by date (bias/zero and flat images only)
+              sqlite3 database registry of ingested calibration products
+              created in calibrepo
+        **NOTE: this does not ingest any defects or fringes!**
+        You must do this manually... e.g.,
+        $ cd calibrepo
+        $ ingestCalibs.py ../repo --calib . --calibType defect --validity 999
+          ../HiTS/MasterCals/defects/2014-12-05/*fits
+processCcd: processedrepo/visit populated with subdirectories containing the
+            usual post-ISR data (bkgd, calexp, icExp, icSrc, postISR)
+
+BASH EQUIVALENTS
+ingest:
     $ ingestImagesDecam.py repo --filetype raw --mode link datafiles
-if doIngestCalibs:
-    $ ingestCalibs.py repo --calib calibrepo --validity 999 datafiles
-if doProcessCcd:
+ingestCalibs:
+    $ ingestCalibs.py repo --calib calibrepo --mode=link --validity 999 datafiles
+processCcd:
     $ cd calibrepo
-    $ processCcd.py ../repo --id visit=visit ccdnum=ccdnum 
-            --output ../outputrepo -C $OBS_DECAM_DIR/config/processCcdCpIsr.py
-            --config calibrate.doAstrometry=False calibrate.doPhotoCal=False
-            --no-versions
+    $ processCcd.py repo --id visit=visit ccdnum=ccdnum
+            --output processedrepo --calib calibrepo
+            -C $OBS_DECAM_DIR/config/processCcdCpIsr.py
+            -C processccd_config.py
+            --config calibrate.doAstrometry=True calibrate.doPhotoCal=True
+    ** note that to run from bash, 'processccd_config.py' must exist
```

Comment:
Added explicit description of what should go here. However, I'm getting a 'no locations for get' error on the `pan-starrs refObjLoader.ref_dataset_name` now when I run this task, and would like your help resolving that. (It can't find gaia either; did these move?)